README.md

```diff
@@ -1,231 +1,131 @@
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+## 🛠 Built With <a name="built-with"></a>
+- Javascript
+- APIs
+- React
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Click Here]()
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+You can have a copy of this repo run on your local machine by just directly cloning it.
 
-### Setup
+To get a local copy up and running follow these simple example steps.
+1. Download the `Zip` or `(git clone)[git@github.com:jkanyi-web/Math_Magicians.git]`
+2. `cd Leaderboard`
+3. Then open in your browser by using `live server`
 
-Clone this repository to your desired folder:
+## Test
 
-<!--
-Example commands:
+For tracking linter errors locally you need to follow these steps:
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+- After cloning the project you need to run this command
+  > npm install
+   `This command will download all the dependencies of the project`
 
-### Install
+- For tracking the linter errors in HTML file run:
+  > npx hint .
 
-Install this project with:
+- For tracking the linter errors in CSS file run:
+  > npx stylelint "**/*.{css,scss}"
 
-<!--
-Example command:
+  - And For tracking the linter errors in JavaScript file run:
+  > npx eslint .
 
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+### Prerequisites
 
-### Deployment
+- Basic knowledge of HTML, CSS, and Javascript
+- Basic knowledge of Git and Github
+- Basic knowledge of React
+- Basic knowledge of APIs
 
-You can deploy this project using:
+To get the output of our project make sure you have google Chrome, Mozila or any other browser installed.
 
-<!--
-Example:
+### Install 
 
-```sh
+> In order to install this project please go through the following steps:
 
-```
- -->
+- Clone the repository
+- Run `npm install`
+- Run `npm start` to start the server
+- Run `npm run build` to build the project
+- Run `npm run dev` to run the project
```

Comment:
- [ ] Since you are using `Vite` and you specified in the _**script**_ section of the package.json that `npm run dev` is used to run the project, it would be nice to remove the instruction for using `npm run start` because it is not specified in the _**script**_ section of the **_package.json_**. 👍